### ISSUE TITLE
Application Crashing React Native v0.55.4 react-native-navigation alpha v2

### ISSUE THREAD

**Issue by Kielan** [wix/react-native-navigation#3705]
### Issue Description

I'm not sure how to ensure my app has been registered at the top level, or whether I am making a user mistake or a dependency versioning installation is not at working parity with v2. Is there a step on xCode to ensure this?

### Steps to Reproduce / Code Snippets / Screenshots
<img width="408" alt="screen shot 2018-08-07 at 1 19 56 am" src="https://user-images.githubusercontent.com/4452059/43763753-6cf4563e-99e0-11e8-9bc0-3fd380bfbe7c.png">

application to reproduce
https://github.com/Kielan/OnyxWalletReactNative

---
### Environment
* React Native Navigation version: 0.55.4
* React Native version: 0.55.4
* React version: 16.4.1
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Alpha on ios simulator

**Comment by Lokendra-rawat** [comment]
Try `2.0.2452`

**Comment by Kielan** [comment]
@Lokendra-rawat I have tried both "^2.0.2425" and "^2.0.2425" though the error message is a little bit different. I also noticed in a working copy where react-native-navigation is using specifically
"babel-preset-react-native": "^5",
which I have also tried, the error is the same 
`cannot Read property 'bindings' of null`
however. the working repo copy remains up to date on the link in the original post. The error is also at the React Bridge so I cannot access deeper into the error stack (to my knowledge)

<img width="369" alt="screen shot 2018-08-07 at 11 11 11 am" src="https://user-images.githubusercontent.com/4452059/43794360-e3487ef8-9a32-11e8-9455-6d793eff6106.png">
